config/locales/de.yml

```diff
@@ -27,6 +27,7 @@ de:
     de: Deutsch
     fr: Français
     hi: 'हिंदी'
+    ch: 中文
```

Comment:
I looked up the language code for Chinese and it's actually `zh`.

Also what do you think of specifying whether it's Simplified or Traditional?


Chinese | zh
Chinese (Simplified) | zh-Hans
Chinese (Traditional) | zh-Hant

Reply:
@julianguyen sure I'll do that!
I also need separate yml files (like ch.yml) for the three right?

Reply:
Yeah you'll want to create the following [here](https://github.com/ifmeorg/ifme/tree/master/config/locales):
* zh.yml
* devise.zh.yml
* devise_invitable.zh.yml
* zh.yml
* kaminari.zh.yml

Not sure if you've already seen our [Translations guide](https://github.com/ifmeorg/ifme/wiki/Translations)!

Reply:
@julianguyen Thanks! I'll make changes and submit the final PR in a day or two!